src/org/mirah/tool/mirah_compiler.mirah

```diff
@@ -164,7 +164,9 @@ class MirahCompiler implements JvmBackend
 
     sorted_asts.each do |node: Node|
       begin
+        node.accept(AstChecker.new,nil) if AstChecker.enabled 
```

Comment:
whitespace. Also, you could extract this into a static method on AstChecker. eg `AstChecker.maybe_check(node)`